website_event_filter_city/tests/test_ui.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2016 Tecnativa - Jairo Llopis
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.tests.common import HttpCase
+
+
+class UICase(HttpCase):
+    def test_ui_website(self):
+        """Test frontend tour."""
+        self.phantom_js(
+            url_path="/event",
+            code="odoo.__DEBUG__.services['web_tour.tour']"
+            ".run('website_event_filter_city')",
+            ready="odoo.__DEBUG__.services['web_tour.tour']"
+            ".tours.website_event_filter_city.ready",
+            login="demo",
+        )
```

Comment:
You could simplify this:
```suggestion
        self.start_tour("/event", "website_event_filter_city", login="portal")
```
Also, adding the `login="portal"` avoids the 404 error you are getting.